### PR TITLE
Remove password from 'Run with Other Students' to allow SSO users.

### PR DESCRIPTION
**Pull request by jsimpson-cc**
[#152206081]

### CODE REVIEW

**Comment by jsimpson-cc** [comment]
@pjanik I'm adding you as reviewer because it looks like you've worked with this area of the code in the past. Please review. Thanks.

Some comments: It might be better to pass an array of integer id rather than the array of object hash { id: <id> } but I don't know if it makes sense to refactor the implementations of the "services" object methods or if any callers rely on the format of `attribute :students, Array[Hash]`
Also there has been some debate over the PT story whether to require password for non-SSO collaborators. If that requirement changes it would make sense to continue to pass the hash. (And in the future could add any other attributes for selecting/filtering on collaborators.)

**Comment by jsimpson-cc** [comment]
Adding @scytacki as reviewer.

File: config/environments/development.rb [hunk 1]
```diff
@@ -51,7 +51,7 @@
     Bullet.enable = true
     # Bullet.bullet_logger = true
     Bullet.rails_logger = true
-    Bullet.add_footer = true
+    # Bullet.add_footer = true
```
Comment (pjanik) [inline config/environments/development.rb]: Is it an intended change?

Comment (jsimpson-cc) [reply config/environments/development.rb]: Do people actually like having this bullet footer in the UI? 
Often it covers half of my screen, then I bring up a js console and there's no room for anything. 
I'll revert this if people prefer having it there.

Comment (scytacki) [reply config/environments/development.rb]: @jsimpson-cc can you revert it here, and make a new PR that uses an environment variable for it? I think the default should be on, but with an environment variable you can easily turn it off.

**Pull request merged**